code/Items/Base/Carriable.cs

```diff
@@ -43,14 +44,9 @@ public new Player Owner
 	public virtual string SlotText => string.Empty;
 
 	/// <summary>
-	/// Instructional text that will display next to a Primary Attack glyph.
+	/// TODO.
 	/// </summary>
-	public virtual string PrimaryAttackHint => string.Empty;
-
-	/// <summary>
-	/// Instructional text that will display next to a Secondary Attack glyph.
-	/// </summary>
-	public virtual string SecondaryAttackHint => string.Empty;
+	public virtual List<UI.BindingTip> BindingTips => new();
```

Comment:
I don't like the class name `UI.BindingTip`, any ideas? I want to avoid `hint` since `hint` means something else in this project.